README.md

```diff
@@ -1,13 +1,299 @@
-# Technology Stack [![Join the chat at https://gitter.im/dwyl/chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dwyl/chat/?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# Technology Stack
 
-This document describes the full technology stack we are using in dwyl.
 
-If you have ***any questions*** please
-[***ask***](https://github.com/dwyl/technology-stack/issues)
+## Why?
+
+As a ***development team***, <br />
+it's ~~good~~ _essential_ to **be _unambiguous_**
+about the **technology/tools** we use for our work. <br />
+**So that _everyone_** is **clear** what we need to learn/use
+to deliver **outstanding results**!
+
+> _If **anything** is **unclear** or you have **any questions** please_
+[***ask***](https://github.com/dwyl/technology-stack/issues).
+
+#### Why Consider a "New Stack" when we _Already_ Good with the "Old" One...?
+
+Our _reasoning_ for
+_considering_ an alternative approach/stack for building web apps
+was fueled by our
+[_curiousity_](https://www.goodreads.com/quotes/tag/curiosity)
+and
+["_shoshin_"](https://en.wikipedia.org/wiki/Shoshin). <br />
+
+> "_The important thing is not to **stop questioning**. **Curiosity** has its own reason for existence._" ~ Albert Einstein <br /> ~
+"Old Man's Advice to Youth: '**Never Lose** a Holy ***Curiosity***.'"
+LIFE Magazine (2 May 1955) p. 64”
+
+In November 2016 we (_once again_) **questioned our _assumptions_**,
+***re-examined*** and
+[***surveyed***](https://github.com/dwyl/learn-elm/issues/10)
+the "landscape" of "_emerging trends_" in web app development.
+We were ~~pleasantly surprised~~ ***delighted*** to see the _amazing progress_
+made by the people in the Elixir, Phoenix and Elm communities!
+These technologies are set to "***take off***" in 2017 and we are _excited_
+to be _sharing_ the technical/competitve advantage with our clients!
+
+## What?
+
+This document + diagram _describes_ the full "**PETE**" Technology Stack
+we use for dwyl products and services (clients).
+
+Each element in our stack was _carefully_ selected based
+on it's individual merits. <br />
+When _assembled_ into a seamless "machine",
+the stack _unrivaled_ for developer productivity and word-class quality!
```

Comment:
Is it supposed to be world-class quality? And if it is word-class, how do I interpret this sentence?